README.md

```diff
@@ -1,6 +1,15 @@
 UI Application for camptocamp.org v6
 ====================================
 
+Requirements
+------------
+
+ * Python 2.7
+ * Node (4.0 or higher)
```

Comment:
Version v0.10.29 is the one we are having problems with. I know that it works with Node 4.0. But we can take a lower version if you tell me which one is ok. :)